exotic/exotic.py

```diff
@@ -1348,9 +1348,12 @@ def image_alignment(imagedata, roi=1):
 def get_pixel_scale(hdul, file, header, pixel_init):
     if file:
         for i in range(len(hdul[0].header)):
-            if hdul[0].header['COMMENT'][i].split(' ')[0] == 'scale:':
-                imscalen = float(hdul[0].header['COMMENT'][i].split(' ')[1])
-                break
+            try:
+                if hdul[0].header['COMMENT'][i].split(' ')[0] == 'scale:':
+                    imscalen = float(hdul[0].header['COMMENT'][i].split(' ')[1])
+                    break
+            except:
+                imscalen = -1
```

Comment:
any reason you don't want to make it a string like "N/A" instead?

Reply:
This variable is required to be a float otherwise it throws more errors below